shared/chat/conversation/attachment-popup/index.native.js

```diff
@@ -7,20 +7,15 @@ import {formatTimeForPopup} from '../../../util/timestamp'
 
 import type {Props} from './'
 
-const AttachmentView = ({isZoomed, onToggleZoom, messageState, path}: {isZoomed: boolean, onToggleZoom: () => void, path: ?string}) => {
-  if (path) {
-    return <NativeImage resizeMode='contain' source={{uri: `file://${path}`}} style={{alignItems: 'center', flexGrow: 1, justifyContent: 'center'}} />
-  } else {
-    return (
-      <Box style={styleContentsCenter}>
-        <ProgressIndicator style={{width: 48}} white={true} />
-      </Box>
-    )
-  }
-}
+const AttachmentView = ({isZoomed, onToggleZoom, messageState, path, previewSize}: {isZoomed: boolean, onToggleZoom: () => void, path: ?string, previewSize: ?{width: number, height: number}}) => (
+  <Box style={{...globalStyles.flexBoxCenter, flex: 1}}>
+    {!!path && <NativeImage resizeMode='contain' source={{uri: `file://${path}`}} style={{alignItems: 'center', flexGrow: 1, justifyContent: 'center', width: previewSize ? previewSize.width : 100, height: previewSize ? previewSize.height : 100}} />}
```

Comment:
This no longer expands larger images to the full screen extents:

![simulator screen shot apr 27 2017 2 39 00 pm](https://cloud.githubusercontent.com/assets/16893/25505602/62f196fe-2b57-11e7-9fe9-4600cbfc1c41.png)